### PR TITLE
M2922 Rifle and M41A Rifle addition

### PR DESCRIPTION
**Pull request by Cameron653**
Adds the M2922 rifle and enables the M41A rifle.

M2922 created by SilencedMP5A5

- Adds both weapons to the automatic weapons crate
- Adds in a M2922 riffle (10 shots, 25 damage per shot. Semiautomatic)
- Enables M41A rifle (30 shots, 15 damage per shot. Semiautomatic/3 round burst)

Picture of weapons and  the ammo they both use. Both L and R hand sprites: 
https://i.gyazo.com/6c721680fd3d4963e49f9343119241c7.png
(Ignore the north facing sprite having black pixels on the characters back. This has been fixed.)
Edit: M2922 magazine sprite switched to SVD mag sprite: https://i.gyazo.com/55a36440c727b882804ef0186434a4c0.png

### CODE REVIEW

**Comment by SpadesNeil** [comment]
We also have better magazines _and_ sound effects that could be used here.

**Comment by Cameron653** [comment]
Any suggestions on which ones to use? I chose the 5.56 magazine sprite since it's already sprited and listed as "5.56" in the game. Also, the SVD sprite does look better than the kurz sprite, so I'll use that, but I have no idea for the sounds. I just chose the ones I thought would best fit the guns.

File: code/modules/vore/fluffstuff/custom_guns_vr.dm [hunk 1]
```diff
@@ -561,7 +561,7 @@
 	projectile_type = /obj/item/projectile/bullet/rifle/a556mm
 
 /obj/item/projectile/bullet/rifle/a556mm
```
Comment (SpadesNeil) [inline code/modules/vore/fluffstuff/custom_guns_vr.dm]: Remove this. This alters damage values for all 5.56 rifles. The Pulse Rifle shouldn't use this bullet anyway.

File: code/modules/vore/fluffstuff/custom_guns_vr.dm [hunk 2]
```diff
@@ -483,6 +544,29 @@
 /obj/item/ammo_magazine/battlerifle/empty
 	initial_ammo = 0
 
+//---------------- M41A -----------------
```
Comment (SpadesNeil) [inline code/modules/vore/fluffstuff/custom_guns_vr.dm]: This all needs to go away. We already have this ammo in game and you're overriding it.

**Comment by SpadesNeil** [comment]
What kind of niche does the M2922 fill? Is there any reason it can't just use the actual SVD magazine and ammo?

File: code/modules/vore/fluffstuff/custom_guns_vr.dm [hunk 3]
```diff
@@ -450,8 +507,12 @@
 	max_ammo = 30
 	mag_type = MAGAZINE
 
+/obj/item/ammo_magazine/stg/small
```
Comment (SpadesNeil) [inline code/modules/vore/fluffstuff/custom_guns_vr.dm]: Why?

**Comment by SpadesNeil** [comment]
Ok there's way too many problems with this. Closing this.

**Pull request closed without merging**